### PR TITLE
remove hidden files and dirs

### PR DESCRIPTION
**Pull request by pzurek**
Just to match the default behaviour of the tree tool this PR removes all hidden files and directories.
I would be quite happy to add a support for a flag bringing hidden files back.

### CODE REVIEW

File: tree/main.go [hunk 1]
```diff
@@ -75,3 +77,14 @@ func visit(path, indent string) (dirs, files int, err error) {
 	}
 	return dirs + 1, files, nil
 }
+
+func removeHidden(files []string) []string {
+	clean := []string{}
```
Comment (campoy) [inline tree/main.go]: please use

```go
var clean []string
```

File: tree/main.go [hunk 2]
```diff
@@ -75,3 +77,14 @@ func visit(path, indent string) (dirs, files int, err error) {
...
+	for _, f := range files {
+		if strings.HasPrefix(f, ".") {
```
Comment (campoy) [inline tree/main.go]: maybe use

```go
if f[0] != '.' {
   clean = append(clean, f)
}
```

that way the code is shorter and faster (`HasPrefix` is overkill here)

**Review by campoy — COMMENTED** [review]
thanks for the contribution!
you might need to rebase

**Review by campoy — APPROVED** [review]

**Comment by campoy** [comment]
Hi @pzurek,

In order to merge your PR I'd need you to sign this CLA.
Sorry, I thought @googleblot would have asked you automatically!

https://cla.developers.google.com/

**Comment by pzurek** [comment]
@campoy I've signed the CLA now, not sure if @googlebot is going to pick it up somehow automatically.

**Comment by campoy** [comment]
somehow I didn't see that you signed it before ... even though apparently you did long time ago!
sorry :)

**Comment by campoy** [comment]
can you please rebase the PR?

**Comment by campoy** [comment]
Merged, thanks!

**Pull request merged**